common/diplomatic_actions/15_customs_union.txt

```diff
@@ -134,24 +134,50 @@
 				limit = {
 					has_variable = total_customs_union_payment_positive
 				}
-				change_variable = {
-					name = total_customs_union_payment_positive
-					add = {
-						subtract = scope:target_country.var:previous_minting_cost
+				if = {
+					limit = {
+						scope:target_country = {
+							has_variable = previous_minting_cost
+						}
+					}
+					change_variable = {
+						name = total_customs_union_payment_positive
+						add = {
+							subtract = scope:target_country.var:previous_minting_cost
+							add = scope:target_country.imperia_weekly_customs_union_minting_cost
+						}
+					}
+				}
+				else = {
+					change_variable = {
+						name = total_customs_union_payment_positive
 						add = scope:target_country.imperia_weekly_customs_union_minting_cost
 					}
 				}
 			}
 			else = {
-				set_variable = {
-					name = total_customs_union_payment_positive
-					value = {
-						add = {
-							subtract = scope:target_country.var:previous_minting_cost
-							add = scope:target_country.imperia_weekly_customs_union_minting_cost
+				if = {
```

Comment:
You could've used `else_if = {` here, and then an `else = {` later. Not sure what gain is from this approach :p